hack/update-godep-licenses.sh

```diff
@@ -112,6 +112,19 @@ process_content () {
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+# Check bash version
+if ((${BASH_VERSINFO[0]}<4)); then
```

Comment:
We should add here a test if the user runs `bash` otherwise this will result in another error:
```
zsh: bad math expression: operand expected at `<4'
````

e.q. 

```

if [[ "$SHELL" != */bash ]]
    echo "It's suggested that you run a bash"
    echo "Ensure you are up to date on the following packages:"
    echo "$ brew install md5sha1sum bash jq"
fi
```